plugins/tileacq.py

```diff
@@ -143,6 +137,25 @@ def _on_streams_change(self, _=None):
         for s in ss:
             for va in self._get_settings_vas(s):
                 va.subscribe(self._update_exp_dur)
+                va.subscribe(self._memory_check)
+
+    def _unsubscribe_vas(self):
+        tab = self.main_app.main_data.tab.value
+        ss = self._get_live_streams(tab.tab_data_model)
+
+        # Unsubscribe to all relevant setting changes
+        for s in ss:
+            for va in self._get_settings_vas(s):
+                va.unsubscribe(self._update_exp_dur)
+                va.unsubscribe(self._memory_check)
+
+        self.nx.unsubscribe(self._update_exp_dur)
```

Comment:
You don't need to unsubscribe from the `.nx`, `.ny` and `.overlap`, as they are VAs which are only show in the dialog. So they will only be updated when it's visible. => they can stay in `__init__()`